### PR TITLE
bottom to up button

### PR DESCRIPTION
**Pull request by harishpansari**
This problem is solved and show the icon.
Thanks

### CODE REVIEW

File: education.html [hunk 1]
```diff
@@ -5,8 +5,10 @@
     <title>John Doe | Education</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="keywords" content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
-    <meta name="description" content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+    <meta name="keywords"
+        content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
+    <meta name="description"
```
Comment (smaranjitghose) [inline education.html:10]: Please undo all indentation and linting related changes.

File: education.html [hunk 2]
```diff
@@ -5,8 +5,10 @@
     <title>John Doe | Education</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="keywords" content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
-    <meta name="description" content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+    <meta name="keywords"
+        content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
```
Comment (smaranjitghose) [inline education.html:9]: Please undo all indentation and linting related changes.

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: education.html (hunk 2 above)
Comment (harishpansari) [reply education.html:9]: Nothing is changing in code.
only changed in "format document"
and it's work.

Comment (smaranjitghose) [reply education.html:9]: I am aware of that 😅. Irrespective of that these sections will be counted as changed in a commit and will give rise to merge conflict for other PRs

**Comment by harishpansari** [comment]
..

**Pull request closed without merging**